README.md

```diff
@@ -2,12 +2,18 @@
 
 # Project Name
 
+Portfolio Mobile Version
+
```

Comment:
- [OPTIONAL] I believe it would be better if you remove the `# Project Name` title and replace it with `# Portfolio` :recycle: . This way it is more clean.